### PR TITLE
add dummy seed data, also one migration to add title column

### PR DESCRIPTION
**Pull request by cbandrow-zz**
### What it do:

Adds some seeded data. One default folder and one default URL. 
Also does a migration to add a new column for Title data. 

### what next:

Hooking up server goodness to Express!

@lauraturk 👍

### CODE REVIEW

**Comment by lauraturk** [comment]
closes #10 

Looks good. Good catch on adding the title column to the database, total miss on my part.

Remember: We're going to do the url shortening on the server side. It will be something we should look into a little bit more.

File: db/seed/dev/folders.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return knex('urls').del()
+    .then(() => knex('folders').del())
+    .then(() =>{
+      return Promise.all([
+        knex('folders').insert({
+          folder_name: 'default'
+        }, 'id')
+        .then(folder =>{
+          return knex('urls').insert({
+            original_url: 'www.idontknowwhattodowithmylife.com',
```
Comment (lauraturk) [inline db/seed/dev/folders.js:13]: this website is hilarious. We may the shortened URL later to reflect however we are shortening it.

File: knexfile.js [hunk 2]
```diff
@@ -4,10 +4,13 @@ module.exports = {
 
   development: {
     client: 'pg',
-    connection: 'postgres://localhost/folders',
+    connection: 'postgres://localhost/jet_fuel',
```
Comment (lauraturk) [inline knexfile.js:7]: much better name!

File: db/seed/dev/folders.js (hunk 1 above)
Comment (cbandrow-zz) [reply db/seed/dev/folders.js:13]: Good point, lets plan on that.

**Pull request merged**